api-tests/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "A base repo for all microservices",
   "main": "index.js",
   "scripts": {
-    "test:awesome": "mocha specs/*/*.js --reporter mochawesome || true",
```

Comment:
updating this has broken the tests, please revert it.

Reply:
Restored accidentally deleted test path!